### DISCUSSION TITLE
chia Stuck @ Syncing 552,064/1,237,410

### DISCUSSION THREAD

**Discussion by becharaSfeir** [Chia-Network/chia-blockchain#9479] [Support]
i a noob in this, so you need to excuse my experience (0) 

i am stuck and do not know what to do, 

any recommendation, i was reading around, and i have checked the netstat, and i see that port 8444 is connecting, 
None of your plots have passed the plot filter yet. But you are currently farming 202.703 GiB

i have 2 plots 
any suggestion, can i do anything or i just remove this software and find something else to do?

**Comment by TomaszWM** [comment]
I have the same issue  chia have stuck syncing at 516.511/1.252.745
